numpy/__init__.py

```diff
@@ -357,6 +357,26 @@ def _mac_os_check():
                 raise RuntimeError(msg)
     del _mac_os_check
 
+    def _win_os_check():
+        """
+        Quick Sanity check for Windows OS: look for fmod bug issue 16744.
+        """
+        try:
+            a = np.arange(13 * 13, dtype=np.float64).reshape(13, 13)
+            a = a % 17  # calls fmod
+            np.linalg.eig(a)
+        except:
```

Comment:
```suggestion
        except Exception:
```
so as not to catch `Ctrl+C`

Reply:
Could also catch actual exception `LinAlgError`